lib/tasks/shf.rake

```diff
@@ -198,6 +198,31 @@ namespace :shf do
     end
   end
 
+  desc 'add member page arg=[filename]'
+  task :add_member_page, [:filename] => :environment do |task_name, args|
+
+    ActivityLogger.open(LOG_FILE, 'SHF_TASK', task_name) do |log|
+
+      filename = args.fetch(:filename) do |_key|
+        log.record('error', 'You must specify a file name')
+        raise 'ERROR: You must specify a file name'
+      end
+
+      # Add html file type if not present
+      filename = filename + '.html' unless filename =~ /.*\.html$/
```

Comment:
If you can make the error message more informative, then this looks good to me.

_(I should have made my previous comment above part of the review.)_